src/bfbs_gen_lua.cpp

```diff
@@ -600,15 +607,15 @@ class LuaBfbsGenerator : public BaseBfbsGenerator {
     // Namespaces are '.' deliminted, so replace it with the path separator.
```

Comment:
I think this should be upstreamed into Namer or BfbsNamer, since there are quite a few languages that emit one file per type.

Reply:
Done.